Cartfile

```diff
@@ -2,7 +2,9 @@ github "ReactiveCocoa/ReactiveSwift" ~> 4.0
 github "ReactiveCocoa/ReactiveCocoa" ~> 8.0
 
 github "Alamofire/Alamofire" ~> 4.7
+github "AckeeCZ/Reqres" ~> 2.3
 
 github "AckeeCZ/ACKategories" ~> 5.6
 
 github "SnapKit/SnapKit" ~> 4.0
+github "sberrevoets/SDCAlertView" ~> 8.0
```

Comment:
Well I think that it might be useful to create our custom `UIViewController` because on many of our projects this dependency would be here just to display errors.